glide.yaml

```diff
@@ -33,6 +33,7 @@ import:
   - tools/clientcmd
 - package: github.com/mcuadros/go-version
 - package: github.com/projectcalico/felix
+  version: 518dbc352494a962e46d50f8bfe72a70c86951bf
```

Comment:
Should this be felix/master/HEAD `07ac30d642849b2e5ebf21a0b0aa72c5b827fe6c`?

Reply:
Nah - because Felix isn't merged yet.  So it's pointing to the commit hash of the branch.  That's why the branch I put up is actually in the Felix repo rather than mine.